### PR TITLE
Added info about /etc/hosts config

### PR DESCRIPTION
**Pull request by dzubenco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -496,6 +496,54 @@ Save the user
 </details>
 
 
+<details>
+<summary><h4>Limit direct access to domains which are used as Nexus Proxy</h4></summary>
+
+#
+Once you setup Proxy repository in Nexus for a specific remote registry, would be better to limit direct access to these registries from a client's machine
```
Comment (irublev) [inline README.md]: , would be better -> it would be better

File: README.md [hunk 2]
```diff
@@ -496,6 +496,54 @@ Save the user
...
+to ensure that each and every command to pull dependencies refers to Nexus repository and uses corresponding Nexus Proxy, but not remote registry from Internet itself.
+
+So, you can create new entries in `/etc/hosts` file as follows (changing this file requires `sudo` privileges) in advance and map necessary domains to `0.0.0.0`, 
+which means that access to these domain names in Internet will be limited. 
+
+Instead, if your configuration of client machine is accurate, your requests to the same remote registries will be done through Nexus Proxy repositories.
```
Comment (irublev) [inline README.md]: accurate seems to me sounding strange, could you please change this?

**Review by irublev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dzubenco) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (dzubenco) [reply README.md]: Done

**Pull request merged**